docs/already-executed-scripts.md

```diff
@@ -0,0 +1,10 @@
+## Installation in a project with already executed scripts
+
+When you already have executed scripts on your project and you want to start using Datamaintain, please follow those steps:
+- Add Datamaintain as a dependency to your project, as described [here](../README.md#installation). 
```

Comment:
I guess the link is not correct because the section `installation` does not exist in README.